pkg/kubelet/util/swap/swap_util.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package swap
+
+import (
+	"bytes"
+	"errors"
+	"k8s.io/apimachinery/pkg/util/version"
```

Comment:
see the pattern of how the imports are organized in pkg/kubelet/cm/container_manager_linux.go, please follow the same here (avoid mixing all the imports into a single section)